notifications_utils/recipients.py

```diff
@@ -405,12 +405,19 @@ def get_international_phone_info(number):
     prefix = get_international_prefix(number)
 
     return international_phone_info(
-        international=(prefix != uk_prefix),
+        international=(prefix != uk_prefix or _is_a_crown_dependency_number(number)),
         country_prefix=prefix,
         billable_units=get_billable_units_for_prefix(prefix)
     )
 
 
+CROWN_DEPENDENCY_RANGES = ['7781', '7839', '7911', '7509', '7797', '7937', '7700', '7829', '7624', '7524', '7924']
```

Comment:
I think we’re accidentally catching a small number of phone numbers which might be mainland UK here. But worst thing that happens is that they only get sent by one provider. So we should look later to see if we can make these ranges more specific. But if you’re happy that this range will catch all crown dependency numbers then I’m happy for us to ship as is.

Reply:
alright, lets do it! ☺️